doc/src/manual/noteworthy-differences.md

```diff
@@ -239,12 +239,12 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Julia uses `nothing` of type `Nothing` to represent a null value, whereas Python uses `None` of type `NoneType`.
   * In Julia, the standard operators over a matrix type are matrix operations, whereas, in Python, the standard operators are element-wise operations. When both `A` and `B` are matrices, `A * B` in Julia performs matrix multiplication, not element-wise multiplication as in Python. `A * B` in Julia is equivalent with `A @ B` in Python, whereas `A * B` in Python is equivalent with `A .* B` in Julia.
   * The adjoint operator `'` in Julia returns an adjoint of a vector (a lazy representation of row vector), whereas the transpose operator `.T` over a vector in Python returns the original vector (non-op).
-  * In Julia, a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch, whereas functions in Python have a single implementation (no polymorphism).
+  * In Julia, a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch, whereas standard functions in Python have a single implementation (no polymorphism), although Python's Standard Library does include single-dispatch capability.
   * There are no classes in Julia. Instead they are structures (mutable or immutable), containing data but no methods.
-  * Calling a method of a class in Python (`a = MyClass(x), x.func(y)`) corresponds to a function call in Julia, e.g. `a = MyStruct(x), func(x::MyStruct, y)`. In general, multiple dispatch is more flexible and powerful than the Python class system.
+  * Calling a method of a class instance in Python (`a = MyClass(x); a.func(y)`) corresponds to a function call in Julia, e.g. `a = MyStruct(x); func(a::MyStruct, y)`. In general, multiple dispatch is more flexible and powerful than the Python class system.
```

Comment:
```suggestion
  * Calling a method of a class instance in Python (`x = MyClass(*args); x.f(y)`) corresponds to a function call in Julia, e.g. `x = MyType(args...); f(x, y)`. In general, multiple dispatch is more flexible and powerful than the Python class system.
```
A couple points: dispatch isn't limited to structs and there's no need for type annotations on function calls.